### DISCUSSION TITLE
How to convert 2D detection data to 3D?

### DISCUSSION THREAD

**Discussion by RustyShackleford73** [SkalskiP/sports#6] [Q&A]
Does the cameras needs calibration to get the extrinsics to perform triangulation?

**Comment by SkalskiP** [comment]
I haven't done any extrinsic camera calibration. My scene was very simple - only one person in the center of the frame. So I did some simple detections post-processing. 

I'll move that question to the discussions Q/A section 💬

**Comment by mcfsantos2010** [comment]
Is it possible to perform pose estimation in real time on several cameras at the same time, synchronizing them? In your scene you use 2 cameras, is it possible to do it with 4 or 6?
https://www.youtube.com/watch?v=AWjKfjDGiYE
